src/plugins/chart_expressions/expression_xy/common/expression_functions/xy_vis_fn.ts

```diff
@@ -47,6 +48,7 @@ export const xyVisFn: XyVisFn['fn'] = async (data, args, handlers) => {
   validateExtent(args.yLeftExtent, hasBar || hasArea, dataLayers);
   validateExtent(args.yRightExtent, hasBar || hasArea, dataLayers);
   validateFillOpacity(args.fillOpacity, hasArea);
+  validateXAxisInterval(args.xAxisInterval);
```

Comment:
Could you, please, add the snapshot test for the error to be thrown by the current validation? Thanks.

Reply:
done